packages/flutter_driver/test/src/real_tests/extension_test.dart

```diff
@@ -543,6 +543,73 @@ void main() {
     expect(await getOffset(OffsetType.center), const Offset(40 + (100 / 2), 30 + (120 / 2)));
   });
 
+  testWidgets('getText', (WidgetTester tester) async {
+    await silenceDriverLogger(() async {
+      final FlutterDriverExtension extension = FlutterDriverExtension((String arg) async => '', true);
+
+      Future<String> getTextInternal(SerializableFinder search) async {
+        final Map<String, String> arguments = GetText(search, timeout: const Duration(seconds: 1)).serialize();
+        final Map<String, dynamic> result = await extension.call(arguments);
+        if (result['isError'] as bool) {
+          return null;
+        }
+        return GetTextResult.fromJson(result['response'] as Map<String, dynamic>).text;
+      }
+
+      await tester.pumpWidget(
+          MaterialApp(
+              home: Scaffold(body:Column(
+                key: const ValueKey<String>('column'),
+                children: <Widget>[
+                  const Text('Hello1', key: const ValueKey<String>('text1')),
```

Comment:
The second const caused an analyzer issue:
```
Analyzing 3 directories...                                      
   info • Avoid const keyword • packages/flutter_driver/test/src/real_tests/extension_test.dart:564:45 • unnecessary_const
```

Reply:
Interestingly, I did not get this info when doing the analyze (Probably omitted the info messages) but I removed it anyway